data/json/mapgen/musicstore.json

```diff
@@ -1,4 +1,41 @@
 [
+  {
+    "id": "musicstore_showpiece",
+    "type": "item_group",
+    "items": [
+      {
+        "distribution": [
+          { "item": "acoustic_guitar", "prob": 100 },
+          { "item": "bagpipes", "prob": 25 },
+          { "item": "banjo", "prob": 60 },
+          { "item": "bone_flute", "prob": 20 },
+          { "item": "clarinet", "prob": 50 },
+          { "item": "flute", "prob": 50 },
+          { "item": "harmonica_holder", "prob": 70 },
+          { "item": "jug_clay", "prob": 20 },
+          { "item": "mp3", "prob": 20 },
```

Comment:
I feel that mp3-player, not being a musical instrument, is out of theme here.

Reply:
Pun intended?